### DISCUSSION TITLE
About the windows configuration in the tauri.conf.json file

### DISCUSSION THREAD

**Discussion by nongyehong** [tauri-apps/tauri#9363] [Q&A] [answered]
![image](https://github.com/tauri-apps/tauri/assets/87641407/5f981216-9a91-4fad-a54b-f77e2c3158dc)

![image](https://github.com/tauri-apps/tauri/assets/87641407/00f39e9d-cc68-444b-9056-a0cb8c579c90)

![image](https://github.com/tauri-apps/tauri/assets/87641407/17d21e15-e958-43f7-861d-d5ac1624819c)

When I set the label to: login, I get this error. If I remove it, I can use drag and drop or other window permissions. Why is this?

**Comment by FabianLars** [comment]
Please share the full migrated.json file. In the context of label specific issues the screenshot is missing the important stuff.

**Reply by nongyehong** [reply]
{
  "$schema": "../gen/schemas/desktop-schema.json",
  "identifier": "default",
  "description": "Capability for the main window",
  "local": true,
  "windows": [
    "main"
  ],
  "permissions": [
    "path:default",
    "event:default",
    "window:default",
    "app:default",
    "resources:default",
    "menu:default",
    "window:allow-create",
    "window:allow-center",
    "window:allow-request-user-attention",
    "window:allow-set-resizable",
    "window:allow-set-maximizable",
    "window:allow-set-minimizable",
    "window:allow-set-closable",
    "window:allow-set-title",
    "window:allow-maximize",
    "window:allow-unmaximize",
    "window:allow-minimize",
    "window:allow-unminimize",
    "window:allow-show",
    "window:allow-hide",
    "window:allow-close",
    "window:allow-set-decorations",
    "window:allow-set-always-on-top",
    "window:allow-set-content-protected",
    "window:allow-set-size",
    "window:allow-set-min-size",
    "window:allow-set-max-size",
    "window:allow-set-position",
    "window:allow-set-fullscreen",
    "window:allow-set-focus",
    "window:allow-set-icon",
    "window:allow-set-skip-taskbar",
    "window:allow-set-cursor-grab",
    "window:allow-set-cursor-visible",
    "window:allow-set-cursor-icon",
    "window:allow-set-cursor-position",
    "window:allow-set-ignore-cursor-events",
    "window:allow-start-dragging",
    "webview:allow-print"
  ]
}

**Reply by nongyehong** [reply]
Both v1 upgraded to v2 projects and newly created v2 projects have this problem

tauri info
[✔] Environment
    - OS: Windows 10.0.26080 X64
    ✔ WebView2: 123.0.2420.65
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.77.1 (7cf61ebde 2024-03-27)
    ✔ cargo: 1.77.1 (e52e36006 2024-03-26)
    ✔ rustup: 1.27.0 (bbb9276d2 2024-03-08)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 20.10.0
    - pnpm: 8.15.4
    - yarn: 1.22.21
    - npm: 10.2.3

[-] Packages
    - tauri [RUST]: 2.0.0-beta.14
    - tauri-build [RUST]: 2.0.0-beta.11
    - wry [RUST]: 0.39.0
    - tao [RUST]: 0.27.0
    - @tauri-apps/api [NPM]: 2.0.0-beta.6
    - @tauri-apps/cli [NPM]: 2.0.0-beta.12

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:6130/
    - framework: Vue.js
    - bundler: Rollup

**Reply by FabianLars** [reply, marked as answer]
```
"windows": [
  "main"
],
```

This means that those permissions will be only applied to windows with the `main` label (this is the default label for the first window that does not have a specific label). In this case you could just add `"login"` to that array and it should work.

**Reply by nongyehong** [reply]
Thanks, or my reading ability is insufficient, before seeing editor-* did not understand, need to concatenate the dynamic window label, such as editor-login

**Reply by nongyehong** [reply]
@FabianLars One more thing I want to ask you is to report the following error when I dynamically create a new window
![image](https://github.com/tauri-apps/tauri/assets/87641407/59a21abb-5162-4077-af5f-b1b361b0bffb)
